src/main/java/pl/javastart/movieclub/web/GenreController.java

```diff
@@ -22,15 +24,25 @@ public class GenreController {
     private final MovieService movieService;
 
     @GetMapping("/genre/{name}")
-    public String getGenre(@PathVariable String name, Model model){
+    public String getGenre(
+            @PathVariable String name,
+            @RequestParam(name = "pageNo", required = false, defaultValue = "1") int pageNo,
```

Comment:
Tutaj tak informacyjnie - jak ustawiasz atrybut defaultValue to required nie jest juz wymagany. Niemniej jednak to zalezy jak sobie ustalisz/ustalicie w projekcie